From: Ben Westover <me@benthetechguy.net>
$QT_DIR is set to the Qt install prefix, which works fine for bundled Qt,
but we're using a system Qt install, so everything is spread out. Correct
the path to the qml and plugin directories for a system install.
---

diff --git a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
index 10e1ea16..8a116690 100644
--- a/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
+++ b/internal/frontend/bridge-gui/bridge-gui/DeployLinux.cmake
@@ -22,8 +22,8 @@ cmake_minimum_required(VERSION 3.22)
 #*****************************************************************************************************************************************************
 
 set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_BINDIR}" "${CMAKE_INSTALL_LIBDIR}" "." "../lib")
-install(DIRECTORY "${QT_DIR}/qml" "${QT_DIR}/plugins"
-        DESTINATION "${CMAKE_INSTALL_PREFIX}")
+install(DIRECTORY "${QT_DIR}/lib/qt6/qml" "${QT_DIR}/lib/qt6/plugins"
+        DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/qt6")
 
 macro( AppendLib LIB_NAME HINT_PATH)
     string(TOUPPER ${LIB_NAME} UP_NAME)
